### DISCUSSION TITLE
Multiselect search is broken on website

### DISCUSSION THREAD

**Discussion by evanwellens** [primefaces/community#378] [PrimeNG] [answered]
All attempts to give focus to the search control on o-muiltiselect are forced o loose focus. Including stackblitz

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've fixed this issue and the fix will be available with 16.8.1 this week

**Reply by ThoSap** [reply]
PrimeNG 16.7.1 is out now
https://github.com/primefaces/primeng/blob/master/CHANGELOG.md#16.7.1
